cmd/meroxa/root/resources/list.go

```diff
@@ -75,21 +74,18 @@ func (l *List) Aliases() []string {
 }
 
 func (l *List) Execute(ctx context.Context) error {
-	var err error
-
 	// What used to be `meroxa list resource-types`
 	if l.flags.Types || l.flags.Type || l.ListTypes {
-		var rTypes []string
-
-		rTypes, err = l.client.ListResourceTypes(ctx)
-
+		rTypes, err := l.client.ListResourceTypes(ctx)
 		if err != nil {
 			return err
 		}
 
 		l.logger.JSON(ctx, rTypes)
 		l.logger.Info(ctx, display.ResourceTypesTable(rTypes, l.hideHeaders))
 
+		output := "\n ✨ View a complete list of available and upcoming resources in the dashboard: https://dashboard.meroxa.io/resources/new"
```

Comment:
@justmisosoup let me know if anything else needs to be updated with this.  we don't have to merge this until right before cutting the CLI release

Reply:
As mentioned in the [implementation plan](https://www.notion.so/Connector-Roadmap-Implementation-Plan-3c3f3be755844f7eac0a56cec81d553b?d=ddb58981c0ed49f5b74bab289ba027d5#251157482d894946a50ed56c5e7f11ee) we can go ahead with this wording and thank you for updating this @janelletavares ! 🙇🏾‍♀️